src/syscall/syscall_libc_wasip1.go

```diff
@@ -397,6 +397,7 @@ func Chmod(path string, mode uint32) (err error) {
 	return Lstat(path, &stat)
 }
 
+// TODO: should this return runtime.wasmPageSize?
 func Getpagesize() int {
 	return libc_getpagesize()
 }
```

Comment:
The WebAssembly page size is a fixed constant, and unless WebAssembly decides to introduce a major backwards compatibility break it's not going to change either. I don't think we need to define this in a single place and reuse everywhere (and if we needed to, libc `getpagesize` would probably be the appropriate place anyway).

In any case, I think it's fine (in fact I'd say just use a numeric constant but a call is fine too).